governance/org-docs/STEERING-COMMITTEE.md

```diff
@@ -0,0 +1,11 @@
+# Steering Committee
+
+This document lists the members of the Organization's Steering Committee. Voting members may be added once approved by the Steering Committee as described in the [charter](./CHARTER.md). By adding your name to this list you are agreeing to abide by all Organization polices, including the [charter](./CHARTER.md), the [code of conduct](./CODE-OF-CONDUCT.md), the [trademark policy](./TRADEMARKS.md), and the [antitrust policy](./ANTITRUST.md). If you are serving on the Steering Committee because of your affiliation with another organization (designated below), you represent that you have authority to bind that organization to these policies.
+
+| **NAME** | **Handle** | **Affiliated Organization** |
+| --- | --- | --- |
+| Stefan Binder | @binste | - |
```

Comment:
```suggestion
| Stefan Binder | @binste | - |
| Joel Ostblom | @joelostblom  | - |
```

Is there a description of the difference between a member of the steering committee and maintainer somewhere? I only saw a difference between a maintainer and contributor in the governance file. I'm likely interested in being part of both, and my initial thoughts would be that it is helpful to outline the differences between the two both for current and future members.

Reply:
There is not per se a description of the difference but there are short definitions of the roles itself. The purpose of the steering committee is described [here](https://github.com/altair-viz/altair/pull/3139/files#diff-b9360dfd18570643af9c585d065c43de923a11720107cefb1887356d4918872bR11) and for a maintainer [here](https://github.com/altair-viz/altair/pull/3139/files#diff-7478290b5dcb8653a2ea160e8827b20cf82228f8db2642a1723b96cbca17f9b7R9) (the part you referenced in your comment). The steering committee is across all projects where as someone could also be a maintainer for just one specific project, e.g. `altair_tiles`, and not care or be involved with any of the other projects. In my understanding, maintainers are the ones who deal with the day-to-day of the development where as the steering committee decides on if a new project should be added/started (for example "should we move `altair_ally` to the `altair-viz` GitHub organization?") and the steering committee is responsible to make decisions in case the maintainers of a project (e.g. `altair_tiles`, `altair`, `altair_viewer`, ...) can't form a consensus. See the chapter on [Decisions](90b5dcb8653a2ea160e8827b20cf82228f8db2642a1723b96cbca17f9b7R13).

Does this help and match with your understanding of the documents? Maybe we can add clearer descriptions in a second step once we ourselves have a better understanding how we live and use this governance framework after a few weeks/months?

Reply:
Thank you @binste , I missed that first link describing the steering committee. After reading that it makes sense, but I do like your clarification here and think that it could probably be worthwhile adding in a second step as you mentioned.